backend/sozisel/lib/sozisel/model/polls/poll_result.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Sozisel.Model.Polls.PollResult do
+  use Sozisel.Model.Schema
+
+  import Ecto.Changeset
+
+  @type t :: %__MODULE__{
+          option_id: String.t()
+        }
+
+  @primary_key false
+  embedded_schema do
+    field :option_id, :string
```

Comment:
What about multichoice? I think polls with multichoice have a more flexible use case. I suggest we can either add the property `allow_multichoice` in the poll schema or make them multichoice by default.

Reply:
I based this logic on strawpoll.com where you can just choose one option. This is a poll, not a form like google forms. But if we need that we can have that in separate PR though.

EDIT: It happens that strawpoll allows for multichoice

And I don't know about separating it in 3 PR's, I would have to have 3 PRs opened and keep track of changes in 3 different branches that would eventually merged together, it would introduces a little mess. The only difference would be the size of PRs when it comes to review but again somebody would have to look through the same amount of code.

Reply:
I based this logic on strawpoll.com where you can just choose one option. This is a poll, not a form like google forms. But if we need that we can have that in separate PR though.

EDIT: It happens that strawpoll allows for multichoice

And I don't know about separating it in 3 PR's, I would have to have 3 PRs opened and keep track of changes in 3 different branches that would eventually merged together, it would introduces a little mess. The only difference would be the size of PRs when it comes to review but again somebody would have to look through the same amount of code.

Reply:
#119 here is an issue to change polls to allow multi choice

Reply:
Ok, that seems reasonable. However, I'm gonna left this pr open so @Przemyslaw5 could take a look at it.